server/src/main/java/org/elasticsearch/snapshots/SnapshotsService.java

```diff
@@ -2353,6 +2355,7 @@ public void onFailure(Exception e) {
 
                         @Override
                         public void onFailure(Exception e) {
+                            logger.debug("failed to complete snapshot deletion [{}] for reason: [{}]", deleteEntry, e.getMessage());
```

Comment:
I had the impression that we don't log stacktrace on debug level to avoid flooding the logs. Stacktraces are for either trace or error+. But I am ok either way.

Reply:
https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#debug says:

> Exceptions logged at `DEBUG` should generally include the complete stack trace.

Flooding the logs shouldn't be a problem here, we do not expect these failures to be common.

Reply:
Apperantly skipping re-reading this doc as part of the onboarding is premature ... thanks!